modules/venat-module-market/src/data/xivapi.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosResponse } from 'axios';
+import { LookupResult } from './result';
+
+export interface XIVAPIItem {
```

Comment:
Should this be split into a cross-module library? Or something to do later as needed?

Reply:
It probably should be, but I don't think we discussed that earlier, so I erred on the side of potential duplication for now. We should discuss what we want to do here as a team.